features/step_definitions/net.rb

```diff
@@ -56,14 +56,16 @@
 end
 
 When /^I open web server via the(?: "(.+?)")? url$/ do |url|
-  @result = BushSlicer::Http.get(url: url, cookies: cb.http_cookies)
+  proxy_opt = {}
+  proxy_opt[:proxy] = env.client_proxy if env.client_proxy
```

Comment:
would this work better? `proxy_opt[:proxy] ||= env.client_proxy`

Reply:
we already know that `proxy_opt[:proxy]` is not set so using `||=` does no service. And we want the option set only if we have the option specified. Otherwise we want that key to not be set at all. Above version should be correct.